core/shared/src/main/scala/org/http4s/QValue.scala

```diff
@@ -74,7 +74,7 @@ final class QValue private (val thousandths: Int) extends AnyVal with Ordered[QV
 
 }
 
-object QValue extends QValuePlatform {
+object QValue {
```

Comment:
Oops?

Reply:
Oh these really aren't needed anymore? my bad :)

Reply:
No, those went away in 0.23.0.  They were in support of a long-deprecated macro, and we cleaned out the oldest deprecated things in #4998.

Reply:
Make sense, I had a momentary panic thinking it was one of the platform traits I introduced :)